public/javascripts/data/login.js

```diff
@@ -0,0 +1,36 @@
+exports.tryLogin = (username, password) => {
+
+    const encryption = require('../encryption')
+    const Database = require('better-sqlite3');
+    const db = new Database('controllers/stories.db');
+
+    // Check for valid user name
+    const userStmt = db.prepare('SELECT user_name FROM users WHERE user_name = (?)');
+    const userNameValue = userStmt.get(username);
+    if(userNameValue == undefined) {
+        console.log(`That user name is not in the database ${username}`)
+        db.close();
+        return false
+    };
+    
+    // Get hashed password from database
+    const passStmt = db.prepare('SELECT encrypted_password FROM users WHERE user_name = (?)');
+    const passValue = passStmt.get(username);
+
+    // Send hashed password to check against user entered password
+    if(encryption.decryptPassword(password, passValue['encrypted_password'])) {
```

Comment:
I'd invert the if check here as well. Remember, you want your main function logic to take place in the main body of the function as much as possible (not indented). This makes your function more clear on its stated intent